core/storage/user/gae_models.py

```diff
@@ -274,6 +274,9 @@ class UserSubscriptionsModel(base_models.BaseModel):
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
     # IDs of feedback thread ids that this user subscribes to.
     feedback_thread_ids = ndb.StringProperty(repeated=True, indexed=True)
+    # IDs of feedback thread ids (new framework) that this user subscribes to.
```

Comment:
You should have a TODO/issue to remove the "(new framework)" and at the same time mark feedback_thread_ids etc as "deprecated, do not use" (see similar fields in ExplorationModel for the convention).

Reply:
Filed #5370